### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by zulcss**
- Fix black warnings
- Fix flake8 warnings

### CODE REVIEW

File: agent/tool-scripts/datalog/prometheus-metrics-datalog [hunk 1]
```diff
@@ -223,18 +251,18 @@ while True:
...
-                del item['help']
+                del item["help"]
                 item["@timestamp"] = time_stamp
                 print(json.dumps(item, sort_keys=True), file=ofp)
                 ofp.flush()
         else:
-            logging.warning("[%s] json.loads(prom2json_out) did not return a"
-                    " list", PROG)
+            logging.warning(
+                "[%s] json.loads(prom2json_out) did not return a" " list", PROG
```
Comment (grafuls) [inline agent/tool-scripts/datalog/prometheus-metrics-datalog:264]: unify string

File: agent/tool-scripts/postprocess/prometheus-metrics-postprocess [hunk 2]
```diff
@@ -67,89 +70,113 @@ def denormalize_metrics(new_item, metrics, docs):
...
     docs = []
     _idx = 0
     for item in metrics_datalog:
         try:
             item = json.loads(item)
         except ValueError:
-            logging.warning("line %d of %s is not a valid json document, skipping"
-                    " the entry", _idx, name)
+            logging.warning(
+                "line %d of %s is not a valid json document, skipping" " the entry",
```
Comment (grafuls) [inline agent/tool-scripts/postprocess/prometheus-metrics-postprocess:111]: unify string

File: agent/tool-scripts/postprocess/prometheus-metrics-postprocess [hunk 3]
```diff
@@ -158,8 +185,12 @@ def process_stdout(metrics_datalog, name):
                 try:
                     denormalize_metrics(new_item, metrics, docs)
                 except InvalidDataFormat as exc:
-                    logging.warning("line %d of %s is an invalid prometheus"
-                            " metrics format, %s", _idx, name, exc)
+                    logging.warning(
+                        "line %d of %s is an invalid prometheus" " metrics format, %s",
```
Comment (grafuls) [inline agent/tool-scripts/postprocess/prometheus-metrics-postprocess:189]: unify string

File: agent/tool-scripts/postprocess/prometheus-metrics-postprocess [hunk 4]
```diff
@@ -170,56 +201,65 @@ def process_stdout(metrics_datalog, name):
...
                     _idx += 1
                     continue
         else:
             try:
                 denormalize_metrics(new_item, metrics, docs)
             except InvalidDataFormat as exc:
-                logging.warning("line %d of %s is an invalid prometheus metrics"
-                        " format, %s", _idx, name, exc)
+                logging.warning(
+                    "line %d of %s is an invalid prometheus metrics" " format, %s",
```
Comment (grafuls) [inline agent/tool-scripts/postprocess/prometheus-metrics-postprocess:219]: unify string

File: agent/tool-scripts/datalog/haproxy-ocp-datalog [hunk 5]
```diff
@@ -189,23 +212,32 @@ class Router(object):
...
-            if all or name == 'metrics':
+            if all or name == "metrics":
                 # We only actively wait() for metrics processes ...
                 returncode = proc.wait()
                 if returncode != 0:
                     errs += 1
                     logging.warning(
-                            "[%s] reaped %s process for %s with error return code:"
-                            " %d", PROG, name, self._pod_name, returncode)
+                        "[%s] reaped %s process for %s with error return code:" " %d",
```
Comment (grafuls) [inline agent/tool-scripts/datalog/haproxy-ocp-datalog:221]: unify string

File: agent/tool-scripts/datalog/pprof-datalog [hunk 6]
```diff
@@ -41,46 +51,59 @@ else:
...
+    logging.error("[%s]: invalid ansible inventory file, '%s': %s", PROG, inv_file, exc)
     sys.exit(1)
 
 try:
     groups = inventory.get_groups_dict()
 except Exception as exc:
-    logging.error("[%s]: error processing groups in ansible inventory file,"
-            " '%s': %s", PROG, inv_file, exc)
+    logging.error(
+        "[%s]: error processing groups in ansible inventory file," " '%s': %s",
```
Comment (grafuls) [inline agent/tool-scripts/datalog/pprof-datalog:86]: unify string

File: agent/tool-scripts/datalog/prometheus-metrics-datalog [hunk 7]
```diff
@@ -60,52 +73,56 @@ def host_n_port(inv_file):
...
-                        key = part.split('=', 1)[1]
+                        key = part.split("=", 1)[1]
                     elif part.startswith("selfsigned="):
-                        selfsigned = part.split('=', 1)[1]
+                        selfsigned = part.split("=", 1)[1]
                     else:
                         logging.warning(
-                                "[%s]: invalid inventory file format at line"
-                                " %d: '%s'", PROG, linenum, line)
+                            "[%s]: invalid inventory file format at line" " %d: '%s'",
```
Comment (grafuls) [inline agent/tool-scripts/datalog/prometheus-metrics-datalog:96]: unify string

**Comment by grafuls** [comment]
Black won't find python scripts with no .py extension based on shebang.
We need to pass paths to scripts to the black command as specified here:
https://github.com/psf/black/issues/491#issuecomment-417111831

**Comment by portante** [comment]
Can you rebase this on the `br69` branch?

File: agent/tool-scripts/postprocess/prometheus-metrics-postprocess (hunk 2 above)
Comment (zulcss) [reply agent/tool-scripts/postprocess/prometheus-metrics-postprocess:111]: done

File: agent/tool-scripts/postprocess/prometheus-metrics-postprocess (hunk 4 above)
Comment (zulcss) [reply agent/tool-scripts/postprocess/prometheus-metrics-postprocess:219]: done

File: agent/tool-scripts/datalog/haproxy-ocp-datalog (hunk 5 above)
Comment (zulcss) [reply agent/tool-scripts/datalog/haproxy-ocp-datalog:221]: done

File: agent/tool-scripts/datalog/prometheus-metrics-datalog (hunk 7 above)
Comment (zulcss) [reply agent/tool-scripts/datalog/prometheus-metrics-datalog:96]: done

**Pull request closed without merging**